src/main/webapp/assets/modules/create-event/createEventController.js

```diff
@@ -7,7 +7,8 @@ techlooper.controller("createEventController", function ($scope, $translate, jso
     }
 
     apiService.createWebinar($scope.webinar).success(function (data) {
-      alert("Successful created Webinar!!");
+      var title = utils.toAscii(data.name);
+      window.location.href = sprintf('#/event-details/'+title+'-'+data.createdDateTime+ '-id');
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'sprintf' is not defined.